tests/core/variant/test_dictionary.h

```diff
@@ -203,9 +183,11 @@ TEST_CASE("[Dictionary] keys() and values()") {
 
 TEST_CASE("[Dictionary] Duplicate dictionary") {
 	// d = {1: {1: 1}, {2: 2}: [2], [3]: 3}
-	Dictionary k2 = build_dictionary(2, 2);
-	Array k3 = build_array(3);
-	Dictionary d = build_dictionary(1, build_dictionary(1, 1), k2, build_array(2), k3, 3);
+	Dictionary k2 = { { 2, 2 } };
+	Array k3 = { 3 };
+	Dictionary d = { { 1, Dictionary({ { 1, 1 } }) },
+		{ k2, Array({ 2 }) },
+		{ k3, 3 } };
```

Comment:
This form is more readable and should pass clang-format (it's already used elsewhere in the file):

```suggestion
	Dictionary d = {
		{ 1, Dictionary({ { 1, 1 } }) },
		{ k2, Array({ 2 }) },
		{ k3, 3 }
	};
```